### PR TITLE
Slight redesign to covid landing page

### PR DESCRIPTION
**Pull request by andysellick**
- make social distancing link more prominent
- move images to the right
- amend design of announcements section
- wrap all in a grey, not black, box

### CODE REVIEW

File: app/assets/stylesheets/views/_covid.scss [hunk 1]
```diff
@@ -10,43 +11,73 @@ $covid-green: #7eff8e;
...
   &:link,
   &:visited {
     color: govuk-colour("white");
   }
   &:focus {
     color: $govuk-text-colour;
   }
+}
+
+.covid__highlight-link {
```
Comment (vanitabarrett) [inline app/assets/stylesheets/views/_covid.scss]: The green SVG on yellow focus is a bit painful - perhaps the focus state should only be on the text? 

Screenshot 2020-03-27 at 18 28 00

Comment (andysellick) [reply app/assets/stylesheets/views/_covid.scss]: Ah yes, I'd forgotten about that. Okay, good point, will take a look.

Comment (andysellick) [reply app/assets/stylesheets/views/_covid.scss]: @vanitabarrett have fixed.

Comment (vanitabarrett) [reply app/assets/stylesheets/views/_covid.scss]: Looks awesome! 👏

**Comment by andysellick** [comment]
@DilwoarH yeah, I think hard coded numbers are the inevitable by-product of high pressure changes like this. We can look at alternatives next week if it's a problem.

**Pull request merged**